pkg/network/server.go

```diff
@@ -977,8 +983,10 @@ func (s *Server) handleMPTDataCmd(p Peer, data *payload.MPTData) error {
 	return nil
 }
 
-// requestMPTNodes requests specified MPT nodes from the peer.
+// requestMPTNodes requests specified MPT nodes from the peer or broadcasts
+// request if peer is not specified.
 func (s *Server) requestMPTNodes(p Peer, itms map[util.Uint256]bool) error {
```

Comment:
Notice that `requestBlocks` naturally spreads the load between peers (requesting different blocks from different ones), but this just asks the same thing from all peers. I don't think it's optimal network-wise.